src/app/zap-templates/zcl/data-model/chip/boolean-sensor-configuration-cluster.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <bitmap name="Feature" type="bitmap32">
+    <cluster code="0x0080"/>
+    <field name="Visual" mask="0x1"/>
+    <field name="Audible" mask="0x2"/>
+    <field name="AlarmSuppress" mask="0x4"/>
+    <field name="SensitivityLevel" mask="0x8"/>
+  </bitmap>
+
+  <bitmap name="AlarmModeBitmap" type="bitmap8">
+    <cluster code="0x0080"/>
+    <field name="Visual" mask="0x1"/>
+    <field name="Audible" mask="0x2"/>
+  </bitmap>
+
+  <enum name="SensitivityEnum" type="enum8">
+    <cluster code="0x0080"/>
+    <item name="High" value="0x0"/>
+    <item name="Standard" value="0x1"/>
+    <item name="Low" value="0x2"/>
+  </enum>
+
+  <cluster apiMaturity="provisional">
+    <name>Boolean Sensor Configuration</name>
+    <domain>Measurement &amp; Sensing</domain>
+    <code>0x0080</code>
+    <define>BOOLEAN_SENSOR_CONFIGURATION_CLUSTER</define>
+    <client tick="false" init="false">true</client>
+    <server tick="false" init="false">true</server>
+    <description>This cluster is used to configure a boolean sensor.</description>
+    <globalAttribute side="either" code="0xFFFD" value="1"/>
+
+    <attribute side="server" code="0x0000" define="SENSITIVITY_LEVEL" type="SensitivityEnum" isNullable="false" writable="true" optional="true">SensitivityLevel</attribute>
```

Comment:
This should probably have min/max values matching the enum range, if we want that to be enforced on writes automatically...  Yes, I know those are not in the spec.

Reply:
Makes sense. I will make the change even though this part is changing in a follow-up PR.

Reply:
Yes, I think we should try to land this with current spec text and then we will have to update it once we get feedback from the ballot. I know this is kind of annoying having to do work, well knowing it needs to be changed later but it seems like this is the process we have to follow 😔

Reply:
The point is: in the spec the min/max is expressed via the type.  In the SDK, min/max are used as part of the implementation, not just to reflect spec, and the type is ignored for range checks...  So this is not a matter of what the spec says; it's a matter of what the implementation here needs.